src/de/hasait/cipa/Cipa.groovy

```diff
@@ -109,6 +112,7 @@ class Cipa implements CipaBeanContainer, Runnable, Serializable {
 		findOrAddBean(StageAroundActivity.class)
 		findOrAddBean(TimeoutAroundActivity.class).withDefaultTimeoutInMinutes(defaultTimeoutInMinutes)
 		findOrAddBean(UpdateGraphAroundActivity.class)
+		findOrAddBean(CipaFileResourceCleanup.class)
```

Comment:
The comment is related to my comment in CipaFileResourceCleanup:
I think adding a param here: boolean enableCleanup = true
and only add the Bean if enableCleanup is true. Then pipelines can disable cleanup.